spec/spec_helper.rb

```diff
@@ -25,17 +26,34 @@ def self.run_all(reporter=nil)
   end
 end
 
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    # mocks.verify_partial_doubles = true
```

Comment:
A lot of specs fail. Not sure if it's worth it considering we're only using `instance_double` once.

Reply:
Complete nonsense ☝️

Reply:
A lot of specs fail, e.g.:
```
  17) RSpec::Rails::SetupAndTeardownAdapter::setup registers teardown hooks in the order setup is received
      Failure/Error: expect(klass).to receive(:after).ordered { |&block| expect(block.call).to eq "foo" }
        #<Class:0x00007fdb8f30ec70> does not implement: after
      # ./spec/rspec/rails/setup_and_teardown_adapter_spec.rb:40:in `block (3 levels) in <top (required)>'

  18) render_template given a symbol converts to_s and delegates to assert_template
      Failure/Error: expect(self).to receive(:assert_template).with("template_name", "this message")
        #<RSpec::ExampleGroups::RenderTemplate::GivenASymbol "converts to_s and delegates to assert_template" (./spec/rspec/rails/matchers/have_rendered_spec.rb:23)> does not implement: assert_template
```

Reply:
There is a method that can surpress verification for specific spec's that need it.